grammar/ASR.asdl

```diff
@@ -233,6 +234,7 @@ ttype
     | Character(int kind, int len, expr? len_expr, dimension* dims)
     | Logical(int kind, dimension* dims)
     | Set(ttype type)
+    | Tuple(ttype type)
```

Comment:
Let's do it like this:
```suggestion
    | Tuple(ttype* type)
```
This specifies a type for each element of the tuple. A tuple in Python is always a constant (although it can be created at runtime). We'll impose a limitation that the type has to be known at compile time. The type consists of a list of types for each element. So you know both the length and the types at compile time.

Reply:
Good idea!